test/e2e/specs/editor/various/block-editor-keyboard-shortcuts.spec.js

```diff
@@ -0,0 +1,220 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+async function addTestParagraphBlocks( { editor, page } ) {
+	await test.step( 'add test paragraph blocks', async () => {
+		await editor.canvas
+			.getByRole( 'button', { name: 'Add default block' } )
+			.click();
+		await page.keyboard.type( '1st' );
+		await page.keyboard.press( 'Enter' );
+		await page.keyboard.type( '2nd' );
+		await page.keyboard.press( 'Enter' );
+		await page.keyboard.type( '3rd' );
+	} );
+}
+
+test.describe( 'Block editor keyboard shortcuts', () => {
```

Comment:
Nit: I think a lot of these tests can be grouped into one to speed up the process. Especially we use mostly the same setup for each test, and each test only does one assertion. It's not a requirement for this migration though!

Reply:
We might need to adjust `test shortcuts handling through portals in the same tree` tests, but it should be doable.

What's the preferred method for having multiple checks in a single file, `test.step` or custom failure messages?

Reply:
> What's the preferred method for having multiple checks in a single file, `test.step` or custom failure messages?

Either! Whichever looks the best to you 😉.

Reply:
Let's skip this for now. The whole suite runs in less than 10s.

I thought I could use `editor.setContent` to reset to the content state, but https://github.com/WordPress/gutenberg/pull/57422#discussion_r1440179645 will complicate tests.